### PR TITLE
Monitoring: Fix environment variables for VPC tests.

### PR DESCRIPTION
**Pull request by busunkim96**
Follow up to #8295.

CC @steinwaywhw

### CODE REVIEW

**Comment by steinwaywhw** [comment]
Hi @busunkim96 sorry I miscommunicated in our last thread. Could you keep the environment variable names as is without changing them? I believe the VPC team sets "PROJECT_ID" when they run the tests.

**Comment by steinwaywhw** [comment]
I think @bmccutchon knows better about what environment variables the VPCSC team needs.

**Comment by busunkim96** [comment]
I changed it as we also use PROJECT_ID as our default system test project. I'll poke around a bit more to see what the best way to sort this out is. Thanks for the heads up!

**Comment by bmccutchon** [comment]
Yes, we do set PROJECT_ID in VPCSC tests.

**Comment by tseaver** [comment]
Both Container Analysis and Grafeas jobs failed with "Test Fusion Url (This url will not work until build finishes)".  I have tagged Kokoro to re-run the tests.

**Comment by bmccutchon** [comment]
We (VPCSC) would actually prefer it if as many of these tests as possible are **not** skipped on your CI so that we can more easily tell whether a failure is VPCSC-related. (If it's also failing on your CI, it's almost certainly not a VPCSC error.)

**Comment by busunkim96** [comment]
@bmccutchon That sounds good. This also came up for the Asset VPC tests. I'll work on resolving the environment variable/test runs today.

File: monitoring/noxfile.py [hunk 1]
```diff
@@ -122,12 +122,17 @@ def system(session):
...
     # Additional setup for VPCSC system tests
-    env = {
-        "PROJECT_ID": "secure-gcp-test-project-4",
-        "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT": os.environ.get(
-            "PROJECT_ID"
-        ),
-    }
+    if os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC") == "true":
+        env = {
+            "PROJECT_ID": "secure-gcp-test-project-4",
```
[the remark this answers is not in the document]
Comment (bmccutchon) [reply monitoring/noxfile.py]: Yes, it is.

File: monitoring/tests/system/test_vpcsc.py [hunk 2]
```diff
@@ -28,6 +28,8 @@
 from google.cloud.monitoring_v3 import enums
 
 PROJECT_INSIDE = os.environ.get("PROJECT_ID", None)
+if len(PROJECT_INSIDE) == 0:
+    PROJECT_INSIDE = None
```
[the remark this answers is not in the document]
Comment (bmccutchon) [reply monitoring/tests/system/test_vpcsc.py]: I don't know where this is generated from.

**Comment by bmccutchon** [comment]
Your understanding is correct.

File: monitoring/tests/system/test_vpcsc.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (busunkim96) [reply monitoring/tests/system/test_vpcsc.py]: @steinwaywhw Would you know how this file was generated?

[the remark this answers is not in the document]
Comment (plamut) [reply monitoring/tests/system/test_vpcsc.py]: BTW, if `PROJECT_ID` is not set in environment, and a default value of `None` is assigned to PROJECT_INSIDE, the check above will crash.

It is thus safer to convert falsy values to `None` like this:
```py
if not PROJECT_INSIDE:
    PROJECT_INSIDE = None
```
Or even more concisely, albeit arguably a tad less straightforward to some people:
```py
PROJECT_INSIDE = PROJECT_INSIDE or None
```

**Pull request merged**